### ISSUE TITLE
Run go build with -i by default

### ISSUE THREAD

**Issue by dlsniper** [go-lang-plugin-org/go-lang-idea-plugin#2757]
The plugin should run ` go build -i ` by default as that would give Go a chance to cache compiled packages by default. The option should be possible to be disabled so I guess it should be present in the user configurable section of go tool arguments rather than in another option in the runner.

**Comment by zolotov** [comment]
> The -i flag installs the packages that are dependencies of the target.

**Comment by dlsniper** [comment]
Yes, that's true. Normally the go tool would find and compile only the changed dependencies but there used to be some bugs around it pre-1.5. I think it would be safe now and for complex apps this does make a difference in compile time.
I'm not sure if enabling it by default is a good thing, maybe a notification could be shown and the user allowed to toggle this on (or off), something like: This build is running with caching turned on, click here to turn off. 
What do you think?

**Comment by zolotov** [comment]
I think that there is still not a word about caching. why do you think that `-i` flag relates to cache?
